scripts/dev-e2e.sh

```diff
@@ -140,14 +140,19 @@ init-cluster() {
 
   mkdir -p "$scripts/.tmp"
 
-  TMPDIR="$scripts/.tmp" "$AOHOME/bin/tooling/kind" load docker-image "$IMAGE_NAME" --name armada 2>&1 \
-   | log_group "Loading Docker image $IMAGE_NAME into Armada cluster";
+  if [ "$ARMADA_MASTER" = "localhost" ] ; then
```

Comment:
ARMADA_MASTER is never just "localhost" is it?

usually something like: "armada://localhost:30002"